Add warning about modulus variant to elliptic integral documentation

This just tripped me up. The elliptic integrals in Sympy are defined with the `m` parameter instead of the `k` parameter (`m = k**2`).

See https://en.wikipedia.org/wiki/Elliptic_integral#Notational_variants. We should add some notes to the docstrings to warn about this.

I am not able to build the documentation. It shows this error:
`Makefile:121: warning: overriding recipe for target '_build/cheatsheet/cheatsheet.pdf'`
`Makefile:116: warning: ignoring old recipe for target '_build/cheatsheet/cheatsheet.pdf'`
`rm -rf _build/logo`
`mkdir -p _build/logo`
`python ./generate_logos.py -d`
`INFO:root: File saved: ./_build/logo/sympy-notail.svg`
`INFO:root: File saved: ./_build/logo/sympy-notailtext.svg`
`INFO:root: File saved: ./_build/logo/sympy-notext.svg`
`ERROR:root:rsvg-convert: command not found`
`Makefile:146: recipe for target 'logo' failed`
`make: *** [logo] Error 127`

Can you please help me with this @asmeurer ?

--- a/sympy/functions/special/elliptic_integrals.py
+++ b/sympy/functions/special/elliptic_integrals.py
@@ -23,6 +23,11 @@ class elliptic_k(Function):
     The function `K(m)` is a single-valued function on the complex
     plane with branch cut along the interval `(1, \infty)`.
 
+    Note that our notation defines the incomplete elliptic integral
+    in terms of the parameter `m` instead of the elliptic modulus
+    (eccentricity) `k`.
+    In this case, the parameter `m` is defined as `m=k^2`.
+
     Examples
     ========
 
@@ -96,6 +101,11 @@ class elliptic_f(Function):
     This function reduces to a complete elliptic integral of
     the first kind, `K(m)`, when `z = \pi/2`.
 
+    Note that our notation defines the incomplete elliptic integral
+    in terms of the parameter `m` instead of the elliptic modulus
+    (eccentricity) `k`.
+    In this case, the parameter `m` is defined as `m=k^2`.
+
     Examples
     ========
 
@@ -162,6 +172,11 @@ class elliptic_e(Function):
 
     The function `E(m)` is a single-valued function on the complex
     plane with branch cut along the interval `(1, \infty)`.
+
+    Note that our notation defines the incomplete elliptic integral
+    in terms of the parameter `m` instead of the elliptic modulus
+    (eccentricity) `k`.
+    In this case, the parameter `m` is defined as `m=k^2`.
 
     Examples
     ========
@@ -268,6 +283,11 @@ class elliptic_pi(Function):
 
     .. math:: \Pi\left(n\middle| m\right) =
               \Pi\left(n; \tfrac{\pi}{2}\middle| m\right)
+
+    Note that our notation defines the incomplete elliptic integral
+    in terms of the parameter `m` instead of the elliptic modulus
+    (eccentricity) `k`.
+    In this case, the parameter `m` is defined as `m=k^2`.
 
     Examples
     ========

